apps/web/src/server/applications/pages/case/documentation/applications-documentation.service.js

```diff
@@ -49,16 +49,19 @@ export const getCaseDocumentationFolderPath = (caseId, folderId) => {
  * Get the documents for the current folder
  *
  * @param {number} caseId
- * @param {DocumentationCategory |undefined} documentationCategory
- * @returns {DocumentationFile[]}
+ * @param {number} folderId
+ * @returns {Promise<DocumentationFile[]>}
  */
-export const getCaseDocumentationFilesInFolder = (caseId, documentationCategory) => {
+export const getCaseDocumentationFilesInFolder = (caseId, folderId) => {
```

Comment:
@DavidJWhiteDev , this is actually the biggest change i've made. this function used to take the whole documentationCategory (i.e. folder) as a parameter and now it only takes just the id.

I did this because, when this function is called from the controller, the folder instance is not yet defined, but if it's a problem for what you're doing I can change it back.